samples/js/fb/slurp/src/main.js

```diff
@@ -133,13 +132,19 @@ async function getPhotos(): Promise<List<any>> {
   // Sadly we cannot issue these requests in parallel because Facebook doesn't give you a way to
   // get individual page URLs ahead of time.
   let result = await new List();
-  let url = 'v2.7/me/photos/uploaded?limit=1000&fields=' + query.join(',') +
+  let url = 'v2.8/me/photos/uploaded?limit=1000&fields=' + query.join(',') +
       '&date_format=U';
   while (url) {
     const photosJSON = await callFacebook(url);
     result = await result.append(await jsonToNoms(photosJSON));
-    url = photosJSON.paging.next;
-    seen += photosJSON.data.length;
+    if (photosJSON.paging !== undefined) {
```

Comment:
If using a key generated from v2.8 and calling v2.7 API, paging and data are not returned and this crashes - so trying to protect this a little better.